index.js

```diff
@@ -0,0 +1,36 @@
+document.addEventListener("DOMContentLoaded", () => {
+ 
+    axios.get("https://deckofcardsapi.com/api/deck/new/").then(res => {
+
+        let id = document.querySelector("#deckID");
+
+        id.innerText = res.data.deck_id;
+        axios.get("https://deckofcardsapi.com/api/deck/"+id.innerText+"/shuffle/")
+     
+    }).catch(err => {
+        debugger;
```

Comment:
Having a console.log here is much more acceptable than having a debugger once you submit your lab